databricks_cli/sdk/service.py

```diff
@@ -1114,3 +1117,80 @@ def delete_repo(self, id, headers=None):
         _data = {}
     
         return self.client.perform_query('DELETE', '/repos/{id}'.format(id=id), data=_data, headers=headers)
+
+class ExecutionContextService(object):
```

Comment:
Could you reuse the basic structure for command/context APIs from Tunnel CLI, so that it won't break, once merged? :)
https://github.com/databricks/databricks-cli/commit/6c429d4f581fc99f54cbf9239879434310037cc7

it's removed, so probably it has to be re-added
https://github.com/databricks/databricks-cli/commit/0296117f531e430cd2f819652ec0837dce14ae98